trainingaddon/acceleratoraddon/web/src/de/hybris/training/controllers/cms/CategoryProductsCarouselComponentController.java

```diff
@@ -27,7 +27,11 @@ protected void fillModel(HttpServletRequest request, Model model, CategoryProduc
         String categoryCode = component.getCategoryCode();
 
         final List<ProductData> products = new ArrayList<>();
-        products.addAll(trainingProductFacade.getProductsForCategory(categoryCode));
+
+        List<ProductData> productsFacade = trainingProductFacade.getProductsForCategory(categoryCode);
+        for (int i =0; i<10; i++) {
```

Comment:
This shouldn't be hardcoded, you should have this as an attribute in the component